http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala

```diff
@@ -159,7 +159,7 @@ private[leonardo] object LeoProfile extends MySQLProfile {
       )
     implicit val formattedByMappedColumnType: BaseColumnType[FormattedBy] =
       MappedColumnType.base[FormattedBy, String](
-        _.asString,
+        _.toString,
```

Comment:
why are we doing `toString` instead of `asString`? `toString` is a default function, so it's hard to tell if the value is really intentional..`asString` is a common function we use to make it obvious that the returned value is an intentional string representation that we need to be care about backawrds compatibility etc

Reply:
I had made that change so that `listDisks()` API could return `formattedBy` value in uppercase. But I have reverted that change as we no longer need `formattedBy` in the response.